src/main/java/sirius/biz/charts/PieChart.java

```diff
@@ -0,0 +1,248 @@
+            labelPath.setAttribute(ATTRIBUTE_D,
+                                   String.format("M %f %f L %f %f",
+                                                 pin.getX(),
+                                                 pin.getY(),
+                                                 label.getX(),
+                                                 label.getY()));
+            labelPath.setAttribute(ATTRIBUTE_STROKE, COLOR_BLACK);
+            labelPath.setAttribute(ATTRIBUTE_STROKE_WIDTH, "0.2");
+            labelGroupElement.appendChild(labelPath);
+
+            labelGroupElement.appendChild(createTextElementForLabel(svgElement,
+                                                                    label.getX(),
+                                                                    label.getY() - 1.5,
+                                                                    3.0,
+                                                                    COLOR_BLACK,
+                                                                    textAnchor,
+                                                                    slice.getLabel()));
+            labelGroupElement.appendChild(createTextElementForLabel(svgElement,
+                                                                    label.getX(),
+                                                                    label.getY() + 3.5,
+                                                                    3.0,
+                                                                    COLOR_GRAY,
+                                                                    textAnchor,
+                                                                    slice.formatQuantity()));
+        });
+
+        return svgElement;
+    }
+
+    private Element createPathElementForSlice(Element svgElement,
+                                              double startRadians,
+                                              double endRadians,
+                                              double radius,
+                                              String color) {
+        Element piecePath = createPathElement(svgElement.getOwnerDocument());
+
+        piecePath.setAttribute(ATTRIBUTE_D, assemblePathDefinitionForSlice(startRadians, endRadians, radius));
+        piecePath.setAttribute(ATTRIBUTE_STROKE, COLOR_GRAY);
+        piecePath.setAttribute(ATTRIBUTE_STROKE_WIDTH, "0.1");
+
+        if (Strings.isFilled(color)) {
+            piecePath.setAttribute(ATTRIBUTE_FILL, color);
+        }
+
+        return piecePath;
+    }
+
+    private String assemblePathDefinitionForSlice(double startRadians, double endRadians, double radius) {
+
+        // if the start and end point are precisely the same, we need to add a small offset to avoid rendering issues
+        double delta = endRadians - startRadians;
+        double offset = (Math.abs(delta % TWO_PI) < 2.0e-4) ? 1.0e-4 : 0.0;
```

Comment:
**Optional:** Wouldn't `0.0002` and `0.0001` be more readable and with the same number of characters?

Reply:
That's a matter of taste 🙃 … But here you go: 5d944a5f2fe2c50ef6bfcb0a59cbb7e00af8a206